switch-configuration/config/types/hiRoom

```diff
@@ -1,21 +1,21 @@
 // Room Switch Template -- All conference rooms Conference Center
-TRUNK	ge-0/0/0	cfSCALE-SLOW,cfSCALE-FAST,cfSpeaker,cfInfra,cfAVLAN,cfStaff,cfSigns,HAM_BRIDGE	// Uplink
-TRUNK	ge-0/0/1	cfSCALE-SLOW,cfSCALE-FAST,cfSpeaker,cfInfra,cfAVLAN,cfStaff,cfSigns,HAM_BRIDGE	// Downlink
+TRUNK	ge-0/0/0	hiSCALE-SLOW,hiSCALE-FAST,hiSpeaker,hiInfra,hiAVLAN,hiStaff,hiSigns	// Uplink
+TRUNK	ge-0/0/1	hiSCALE-SLOW,hiSCALE-FAST,hiSpeaker,hiInfra,hiAVLAN,hiStaff,hiSigns	// Downlink
 RSRVD	6
-TRUNK	ge-0/0/8	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/9	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/10	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/11	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/12	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/13	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/14	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/15	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/16	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-TRUNK	ge-0/0/17	cfInfra,cfSCALE-SLOW,cfSCALE-FAST,cfStaff				// AP
-VLAN	cfSigns		4	// ge-0/0/{18-21}
-VLAN	cfAVLAN		6	// ge-0/0/{22-27}
-VLAN	cfSpeaker	2	// ge-0/0/{28,29}
-VLAN	cfCTF		1	// ge-0/0/30
-VLAN	HAM_BRIDGE	1	// ge-0/0/31						// HAM Link N6S to Expo
-VLAN	cfInstall	8	// ge-0/0/{32-39}
+TRUNK	ge-0/0/8	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/9	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/10	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/11	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/12	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/13	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/14	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/15	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/16	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+TRUNK	ge-0/0/17	hiInfra,hiSCALE-SLOW,hiSCALE-FAST,hiStaff				// AP
+VLAN	hiSigns		4	// ge-0/0/{18-21}
+VLAN	hiAVLAN		6	// ge-0/0/{22-27}
+VLAN	hiSpeaker	2	// ge-0/0/{28,29}
+VLAN	hiCTF		1	// ge-0/0/30
+RSRVD	1			// ge-0/0/31 (was HAM_BRIDGE)
```

Comment:
Thanks for cleaning up the HAM bridge. I know that not be required from a few scales now IIRC

Reply:
It's still there in the PCC config files, IIRC, but it makes no sense in the Hilton context since there aren't any separate buildings via routed segments.